zerver/views/registration.py

```diff
@@ -63,6 +63,15 @@ def redirect_and_log_into_subdomain(realm, full_name, email_address,
                                salt='zerver.views.auth')
     return response
 
+def send_initial_pms(user):
+    # type: (UserProfile) -> None
+    content = '''Welcome to Zulip!
+    This is a great place to test formatting, sending, and editing messages :pencil:.
+    Click anywhere on this message to reply! A compose box will open at the bottom of the screen.'''
+
```

Comment:
Let's add another newline after "Welcome to Zulip!"
Also, the current line spacing means the pencil isn't looking great, so I guess let's just remove that. Let's also change the `!` to a `.` in the third line.